src/NearCache.md

```diff
@@ -321,4 +321,5 @@ The Near Cache preloader is a functionality to store the keys from a Near Cache
 The Near Cache preloader stores the keys (not the values) of Near Cache entries in regular intervals. You can define the initial delay via `store-initial-delay-seconds`, e.g., if you know that your hot data set will need some time to build up. You can configure the interval via `store-interval-seconds` which determines how often the key-set will be stored. The persistence will not run continuously. Whenever the storage is scheduled, it will be performed on the actual keys in the Near Cache.
  
 The Near Cache preloader will be triggered on the first initialization of the data structure on the client, e.g., `client.getMap("myNearCacheMap")`. This schedules the preloader, which will work in the background, so your application is not blocked. The storage will be enabled after the loading is completed.
- 
\ No newline at end of file
+
+![image](images/NoteSmall.jpg) ***NOTE:*** *If you run multiple Hazelcast clients with enabled Near Cache preloader on the same machine, you have to configure a unique storage filename for each client or run them from different user directories. If two clients would write into the same file only the first client will succeed. The following clients will throw an exception as soon as the Near Cache preloader is triggered.*
```

Comment:
...multiple Hazelcast clients with Near Cache preloader enabled on the same machine, you have to configure a unique storage filename for each client or run them from different user directories. If two clients write into the same file, only the first client succeeds. The following clients will throw an exception as soon as the Near Cache preloader is triggered.*